client/verta/tests/unit_tests/strategies.py

```diff
@@ -95,3 +98,49 @@ def code_blob_proto(draw) -> Code_pb2.CodeBlob:
         proto.notebook.CopyFrom(draw(notebook_code_blob_proto()))
 
     return proto
+
+
+@st.composite
+def mock_kafka_configs_response(draw) -> Dict[str, Any]:
+    """
+    Provide mocked API result from `api/v1/uac-proxy/system_admin/listKafkaConfiguration`
+    with a single Kafka configuration.
+    """
+    return {
+        "configurations": [
+            {
+                "id": draw(st.integers()),
+                "kerberos": {
+                    "enabled": draw(st.booleans()),
+                    "client_name": draw(st.text()),
+                    "conf": draw(st.text()),
+                    "keytab": draw(st.text()),
+                    "service_name": draw(st.text()),
+                },
+                "brokerAddresses": draw(st.text()),
+                "enabled": draw(st.booleans()),
+                "name": draw(st.text()),
+            }
+        ]
+    }
+
+
+@st.composite
+def mock_kafka_settings(draw) -> KafkaSettings:
+    """Generate a mocked KafkaSettings object with no value for cluster config ID."""
+    topics = draw(st.lists(st.text(min_size=1), min_size=3, max_size=3, unique=True))
```

Comment:
has to be unique to avoid errors thrown when the input topic matches output or error topics.

Reply:
Minor point, but I think the more idiomatic way to write this would be something like

```python
input_topic = draw(st.text(min_size=1))
output_topic = draw(st.text(min_size=1))
error_topic = draw(st.text(min_size=1))
hypothesis.assume(input_topic != output_topic != error_topic)
```

because using the `st.lists(min_size=3, max_size=3)` generator purely as a mechanism to ensure uniqueness feels like...co-opting it for something it's not meant to be.

Reply:
Alright, let's just go with the original.